src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4176,40 +4176,61 @@ .FORWARDHELPCATEGORY Cmdlet
     }
     elseif ($help -ne $null)
     {
-        $customPagerCommand = """"
-        $customPagerCommandArgs = """"
-        $customPagerCommandLine = """"
-
-        if ($env:PAGER)
-        {
-            $customPagerCommandLine = $env:PAGER
+        # By default use more on Windows and less on Linux.
+        if ($IsWindows) {
+            $pagerCommand = 'more.com'
+            $pagerArgs = $null
+        }
+        else {
+            $pagerCommand = 'less'
+            $pagerArgs = '-Ps""Page %db?B of %D:.\. Press h for help or q to quit\.$""'
+        }
+
+        # Respect PAGER environment variable which allows user to specify a custom pager.
+        # Ignore a pure whitespace PAGER value as that would cause the tokenizer to return 0 tokens.
+        if (![string]::IsNullOrWhitespace($env:PAGER)) {
+            if (Get-Command $env:PAGER -ErrorAction Ignore) {
+                # Entire PAGER value corresponds to a single command.
+                $pagerCommand = $env:PAGER
+                $pagerArgs = $null
+            }
+            else {
+                # PAGER value is not a valid command, check if PAGER command and arguments have been specified.
+                # Tokenize the specified $env:PAGER value. Ignore tokenizing errors since any errors may be valid
+                # argument syntax for the paging utility.
+                $errs = $null
+                $tokens = [System.Management.Automation.PSParser]::Tokenize($env:PAGER, [ref]$errs)
+
+                $customPagerCommand = $tokens[0].Content
+                if (!(Get-Command $customPagerCommand -ErrorAction Ignore)) {
+                    # Custom pager command is invalid, issue a warning.
+                    Write-Warning ""Ignoring invalid custom-paging utility command line specified in `$env:PAGER: $env:PAGER""
```

Comment:
This is more that it was not found?  Can we update the message?

Reply:
I can update the message. How about: ""Custom-paging utility command not found. Ignoring command specified in `$env:PAGER: $env:PAGER""

Reply:
that looks better.

Reply:
Done